WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -44,7 +44,9 @@ class GutenbergSettings {
     /// - Returns: true if the post must be edited with Gutenberg.
     ///
     func mustUseGutenberg(for post: AbstractPost) -> Bool {
+        let postIsEmpty = post.content?.isEmpty ?? true
```

Comment:
This is actually a great candidate for adding to `BasePost`.

Also, check out [`BasePost`'s `hasContent() -> Bool` method](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Models/BasePost.h#L43), in case we can use that (not sure).